### PR TITLE
fix install-EasyBuild-develop.sh script w.r.t. vsc-base dependency

### PR DESCRIPTION
**Pull request by boegel**
@besserox: please review?

### CODE REVIEW

**Comment by boegel** [comment]
Hmm, need to look into this:

```
=== Cloning boegel/easybuild-easyconfigs ...
...
=== Checking out the 'develop' branch
fatal: A branch named 'develop' already exists.
```

**Comment by besserox** [comment]
I tried the new version of the script. I forked the vsc repository on github account and the installation was fine. Then when I tried to run EasyBuild, I get the following error:

```
$ eb -h
Traceback (most recent call last):
  File "/home/xavier/tmp/eb/install/easybuild-framework/easybuild/main.py", line 45, in <module>
    from easybuild.tools.build_log import EasyBuildError, init_logging, print_msg, print_error, stop_logging
  File "/home/xavier/tmp/eb/install/easybuild-framework/easybuild/tools/build_log.py", line 38, in <module>
    from vsc.utils import fancylogger
ImportError: No module named vsc.utils

$ echo $PYTHONPATH
/home/xavier/tmp/eb/install/easybuild-easyconfigs:/home/xavier/tmp/eb/install/easybuild-easyblocks:/home/xavier/tmp/eb/install/easybuild-framework:/home/xavier/tmp/eb/install/vsc-base
```

If I use `$root/vsc-base/lib` instead of `$root/vsc-base` in my PYTHONPATH, it seems to work.

File: easybuild/scripts/install-EasyBuild-develop.sh [hunk 1]
```diff
@@ -69,6 +69,7 @@ conflict    EasyBuild
 
 prepend-path    PATH            "\$root/easybuild-framework"
 
+prepend-path    PYTHONPATH      "\$root/vsc-base"
```
Comment (besserox) [inline easybuild/scripts/install-EasyBuild-develop.sh]: It looks like it should be `prepend-path PYTHONPATH "\$root/vsc-base/lib"` here.

Comment (boegel) [inline easybuild/scripts/install-EasyBuild-develop.sh]: You're right, will fix.

I was still testing the updated script myself as well, but bumped into another issue (that I still need to figure out).

**Comment by besserox** [comment]
It looks good to me now. I tested and it works fine.

**Comment by boegel** [comment]
@besserox: thanks for the feedback!

**Pull request merged**